frontend/app/admin/src/views/settings/membership-types/EditMembershipTypeConfigView.vue

```diff
@@ -12,55 +15,43 @@
-            <DateSelection v-model="expireDate" :required="false" :placeholder="$t('admin.settings.membershipTypes.expireDate.placeholder')" :time="{hours: 23, minutes: 59, seconds: 59}" />
+        <STInputBox
+            v-if="type.behaviour === PlatformMembershipTypeBehaviour.Period"
+            :title="$t('admin.settings.membershipTypes.expireDate.title')" error-fields="expireDate"
+            :error-box="errors.errorBox"
+        >
+            <DateSelection
+                v-model="expireDate" :required="false"
+                :placeholder="$t('admin.settings.membershipTypes.expireDate.placeholder')"
+                :time="{hours: 23, minutes: 59, seconds: 59}"
+            />
         </STInputBox>
         <p v-if="type.behaviour === PlatformMembershipTypeBehaviour.Period" class="style-description-small">
             {{ $t('admin.settings.membershipTypes.expireDate.description') }}
         </p>
 
-        <div v-for="(price, index) of prices" :key="price.id" class="container">
-            <hr>
-
-            <template v-if="prices.length > 1">
-                <h2 v-if="index === 0 || !price.startDate">
-                    Standaardprijs
-                </h2>
-                <h2 v-else class="style-with-button">
-                    <div>
-                        Prijs vanaf {{ formatDate(price.startDate) }}
-                    </div>
-                    <div>
-                        <button class="button text" type="button" @click="deletePrice(price)">
-                            <span class="icon trash" />
-                            <span class="hide-smartphone">{{ $t('Verwijderen') }}</span>
-                        </button>
-                    </div>
-                </h2>
-            </template>
-
-            <STInputBox v-if="index > 0 || price.startDate" :title="$t('admin.settings.membershipTypes.period.priceDate')" :error-box="errors.errorBox">
-                <DateSelection :model-value="price.startDate" :required="false" :placeholder="$t('admin.settings.membershipTypes.expireDate.placeholder')" @update:model-value="patchPrice(price, {startDate: $event})" />
-            </STInputBox>
-
-            <div class="split-inputs">
-                <STInputBox v-if="type.behaviour === PlatformMembershipTypeBehaviour.Days || price.pricePerDay" title="Prijs per dag" :error-box="errors.errorBox">
-                    <PriceInput :model-value="price.pricePerDay" placeholder="Prijs per dag" @update:model-value="patchPrice(price, {pricePerDay: $event})" />
-                </STInputBox>
-
-                <STInputBox :title="type.behaviour === PlatformMembershipTypeBehaviour.Days ? 'Vaste prijs' : 'Prijs'" :error-box="errors.errorBox" error-fields="price">
-                    <PriceInput :model-value="price.price" placeholder="Gratis" @update:model-value="patchPrice(price, {price: $event})" />
-                </STInputBox>
-            </div>
-        </div>
+        <PlatformMembershipTypePriceConfigEdit
```

Comment:
Kan je deze componente een duidelijkere naam geven? Meestal gebruik ik 'box' voor dit soort componenten

```suggestion
        <PlatformMembershipTypePriceConfigEditBox
```